tests/protoc.bzl

```diff
@@ -35,6 +35,11 @@ _protoc_wrapper = rule(
     implementation = _protoc_wrapper_impl,
     attrs = {
         "_protoc": attr.label(executable = True, cfg = "exec", default = "@com_google_protobuf//:protoc"),
+        "_cc_toolchain": attr.label(
+            default = Label(
+                "@rules_cc//cc:current_cc_toolchain",  # copybara-use-repo-external-label
```

Comment:
Do we need the `copybara-use-repo-external-label` comment?

Reply:
No, just copied it inadvertently.